cloudwatch-metrics-resque.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.4'
 
-  spec.add_dependency 'aws-sdk-core', '~> 3'
```

Comment:
We require this in `lib/cloud_watch_metrics/util.rb` is that going to break?

Reply:
Ahh crap I didn't check the file with the change in. Will check that in now